tests/countries/test_nepal.py

```diff
@@ -0,0 +1,545 @@
+        name = "Maha Shivaratri"
+        dt = (
+            "2019-03-04",
+            "2020-02-21",
+            "2021-03-11",
+            "2022-03-01",
+            "2023-02-18",
+            "2024-03-08",
+            "2025-02-26",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2016, 2036))
+
+    def test_holi_hilly(self):
+        name = "Holi (Mountain & Hilly)"
+        dt = (
+            "2019-03-20",
+            "2020-03-09",
+            "2021-03-28",
+            "2022-03-17",
+            "2023-03-07",
+            "2024-03-24",
+            "2025-03-13",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2010, 2036))
+
+    def test_holi_terai(self):
+        name = "Holi (Terai)"
+        dt = (
+            "2019-03-21",
+            "2020-03-10",
+            "2021-03-29",
+            "2022-03-18",
+            "2023-03-08",
+            "2024-03-25",
+            "2025-03-14",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2010, 2036))
+
+    def test_nepali_new_year(self):
+        name = "Nepali New Year (Vikram Sambat)"
+        dt = (
+            "2019-04-14",
+            "2020-04-13",
+            "2021-04-14",
+            "2022-04-14",
+            "2023-04-14",
+            "2024-04-13",
+            "2025-04-13",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2010, 2036))
+
+    def test_ram_navami(self):
+        name = "Ram Navami"
+        dt = (
+            "2021-04-21",
+            "2022-04-10",
+            "2023-03-30",
+            "2024-04-17",
+            "2025-04-06",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, (*range(2010, 2016), *(range(2021, 2036))))
+        self.assertNoHolidayName(name, range(2019, 2021))
+        dt = (
+            "2019-04-13",
+            "2020-04-02",
+        )
+        self.assertHolidayName(name, self.workday_holidays, dt)
+        self.assertNoHolidayName(name, self.workday_holidays, range(2010, 2019), range(2021, 2050))
+
+    def test_buddha_jayanti(self):
+        name = "Buddha Jayanti"
+        dt = (
+            "2019-05-18",
+            "2020-05-07",
+            "2021-05-26",
+            "2022-05-16",
+            "2023-05-05",
+            "2024-05-23",
+            "2025-05-12",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2010, 2036))
+
+    def test_id_ul_fitr(self):
+        name = "Id-ul-Fitr"
+        dt = (
+            "2019-06-05",
+            "2020-05-25",
+            "2021-05-14",
+            "2022-05-03",
+            "2023-04-22",
+            "2024-04-11",
+            "2025-03-31",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, self.no_estimated_holidays, range(2010, 2026))
```

Comment:
```suggestion
        self.assertHolidayName(name, self.no_estimated_holidays, range(2010, 2050))
```
More carefully, please.

Reply:
Custom Islamic class has dates upto 2025 only. Afterwards estimated label will be shown so range new range should 2026.?
Should I add new range with estimated holidays as yes from 2026 to 2050 ?

Reply:
And for this purpose, here is `self.no_estimated_holidays`.